backend/de.metas.distribution.rest-api/src/main/java/de/metas/distribution/workflows_api/DistributionEventProcessCommand.java

```diff
@@ -36,6 +54,40 @@ class DistributionEventProcessCommand
 	private DistributionJob changedJob;
 	private HuId _huIdToPick; // lazy
 
+	@Builder
+	public DistributionEventProcessCommand(
+			@NonNull final ITrxManager trxManager,
+			@NonNull final HUQRCodesService huQRCodesService,
+			@NonNull final IHandlingUnitsBL handlingUnitsBL,
+			@NonNull final DDOrderMoveScheduleService ddOrderMoveScheduleService,
+			@NonNull final DistributionJobLoaderSupportingServices loadingSupportServices,
+			@NonNull final InventoryService inventoryService,
+			@NonNull final IUOMConversionBL uomConversionBL,
+			@NonNull final IHUPIItemProductBL hupiItemProductBL,
+			@NonNull final HUAccessService huAccessService,
+			@NonNull final DistributionJob job,
+			@NonNull final JsonDistributionEvent event,
+			final DistributionJob changedJob,
+			final HuId _huIdToPick)
```

Comment:
avoid _ in builder fields

Reply:
that and the changedJOb shouldn't be there, thx!